SECURITY.md

```diff
@@ -0,0 +1,14 @@
+# Security Policy
+
+## Supported Versions
+Due to frequent breaking changes from the HoYoverse/Cognosphere regarding game repair indexes, cache files, update zip files, etc. we can only support any latest Collapse version, be it either from Stable or Preview branch.
+Additionally, due to major changes that happened on the Collapse, we cannot support any version below 1.69.0 and should be updated ASAP to latest version, either with in-build updater, or by reinstalling Collapse Launcher manually (if the built-in launcher fails to update from such old version of Collapse).

```

Comment:
Rephrase: Additionally, due to a major rework in the update infrastructure, any version of Collapse released prior to 1.69.0 **will not be supported** and should be updated ASAP to the latest version. This can be done either through the new built-in updater or by doing a clean install of Collapse.

Reply:
Applied